api/apps/api/src/modules/access-control/projects-acl/project-acl.module.ts

```diff
@@ -0,0 +1,20 @@
+import { Module } from '@nestjs/common';
+import { CqrsModule } from '@nestjs/cqrs';
+import { TypeOrmModule } from '@nestjs/typeorm';
+
+import { UsersProjectsApiEntity } from '@marxan-api/modules/projects/control-level/users-projects.api.entity';
+
+import { ProjectAclService } from './project-acl.service';
+import { ProjectAclController } from './project-acl.controller';
+
+@Module({
+  imports: [
+    CqrsModule,
+    // this entity most likely shouldn't be under `projects`
+    TypeOrmModule.forFeature([UsersProjectsApiEntity]),
+  ],
+  providers: [ProjectAclService],
+  exports: [ProjectAclService],
+  controllers: [ProjectAclController],
+})
```

Comment:
Like for roles, seems we should move the `UsersProjectsApiEntity` under access control module and get rid of relation.

Reply:
also, keeping in-code relation to `users` causes kind of circular relation/dependency. We don't want the access-control to care about particular users, not even to be able to identify them - why would we?

Reply:
The thing is that this entity was used also in a couple of places around the projects directory to be able to do some extended queries to the DB while looking for some particular users. Is this not needed anymore with the ACL-module? I was going to move it, but it involved changing `project-module` and some other files and I wanted to be careful before doing so.

Reply:
We can shift it to another PR but in general yes, `project` itself shouldn't care much. As you wrote, there is a challenge how to make listing projects, for example.